statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1297,23 +1670,66 @@ def resid_acorr(self, nlags=1):
         Parameters
         ----------
         nlags : int
+            The number of lags to use in compute the autocorrelation. Does
+            not count the zero lag, which will be returned.
 
         Returns
         -------
+        acorr : ndarray
+            The autocorrelation for the residuals. The shape is
+            (nlags + 1, neqs, neqs).
         """
         acovs = self.resid_acov(nlags=nlags)
         return _acovs_to_acorrs(acovs)
 
     @cache_readonly
     def resid_corr(self):
-        "Centered residual correlation matrix"
+        """
+        Centered residual correlation matrix
+        """
         return self.resid_acorr(0)[0]
 
-    @cache_readonly
+    @property
+    def sigma_u(self):
+        """
+        Debiased estimate of noise process covariance
+
+        Notes
+        -----
+        Deprecated. Use cov_resid
+        """
+        # Deprecated in 0.10.0, remove in 0.11.0
+        import warnings
+        warnings.warn('sigma_u has been deprecated in favor of cov_resid',
+                      DeprecationWarning)
+        return self.cov_resid
+
+    @property
     def sigma_u_mle(self):
-        """(Biased) maximum likelihood estimate of noise process covariance
         """
-        return self.sigma_u * self.df_resid / self.nobs
+        Debiased estimate of noise process covariance
+
+        Notes
+        -----
+        Deprecated. Use cov_resid
+        """
+        # Deprecated in 0.10.0, remove in 0.11.0
+        import warnings
+        warnings.warn('sigma_u_mle has been deprecated in favor of '
+                      'cov_resid_mle', DeprecationWarning)
+        return self.cov_resid
```

Comment:
Should return `self.cov_resid_mle`?  Is the docstring right?

Reply:
Yes.